app/models/sponsor.rb

```diff
@@ -0,0 +1,4 @@
+class Sponsor < ApplicationRecord
+  validates :company_name, presence:true
```

Comment:
You can make a oneliner for these two validations. Maybe that it is prettier ? :)